input/pagecontent/index.md

```diff
@@ -1,44 +1,62 @@
+
+As the only *language* designed for profiling and IG creation, FSH is unique among these methods. It provides a fast, scalable, and user-friendly path to IG creation and maintenance. Because it is text-based, FSH brings a degree of editing agility not found in graphical tools (such as cutting and pasting, global search and replace, spell checking, etc.) Because it is a HL7 FHIR standard, tooling can be built around FSH with confidence in its stability and continuity.
+
+### About this IG
+
+The FSH IG includes the following information:
+
+1. This page, providing introductory material  _(informative content)_.
+1. [FHIR Shorthand Overview](overview.html) -- Introduction to FSH language and SUSHI (the reference implementation of a FSH compiler) _(informative content)_.
+1. [FHIR Shorthand Language Reference](reference.html) -- The syntax and usage of the FHIR Shorthand language _(formal content)_.
+1. A [Quick Reference Sheet](FSHQuickReference.pdf) under the Downloads menu _(informative content)_.
+
+The following material, useful for learning and applying FHIR Shorthand but not part of the language specification, is found on [FSHSchool.org](https://fshschool.org/):
+
+1. [SUSHI User Guide](https://fshschool.org/docs/sushi/) -- SUSHI ("SUSHI Unshortens ShortHand Inputs") is a reference implementation of an interpreter/compiler for FHIR Shorthand. SUSHI produces [Health Level Seven (HL7®) Fast Healthcare Interoperability Resources (FHIR®)](https://www.hl7.org/fhir/R4/overview.html) profiles, extensions, and other artifacts needed to create FHIR Implementation Guides (IG).
+1. [GoFSH User Guide](https://fshschool.org/docs/gofsh/) -- GoFSH is a tool that turns FHIR artifacts into FSH. Using GoFSH, existing FHIR artifacts or complete Implementation Guides can be transformed into FSH, automatically.
+1. [FHIR Shorthand Tutorials](https://fshschool.org/docs/tutorials/) -- A step-by-step hands-on introduction to producing an Implementation Guide (IG) with FHIR Shorthand and SUSHI.
+1. [FSH Online](https://fshschool.org/FSHOnline/#/) -- A coding playground for FSH, an online environment that allows you to write FSH and convert it to FHIR artifacts, convert FHIR artifacts to FSH, access examples, and share FSH code with others.
+1. [FSH Finder](https://fshschool.org/fsh-finder/) -- A list of public GitHub repositories that contain FSH code, refreshed daily.
+
+Note that the [Language Reference](reference.html) is the formal specification, and if there is any conflict between that and any other written or programmatic materials, the former is considered the source of truth.
```

Comment:
Two questions for your consideration:

1. Should the ANLTR grammar perhaps be its own section in the documentation?
2. (Related to the first question) Should the ANTLR grammar be a formal part of the specification?  Or should we consider it as "informative" since it's actually an implementation of what is defined in the formal spec?  Someone could implement FSH without it (or using a non-ANTLR grammar representation).

Reply:
That's interesting idea. Maybe it is even should be considered part of SUSHI. Let's leave it for now, and make a decision on the next round of changes.